src/main/java/edu/harvard/iq/dataverse/externaltools/ExternalToolServiceBean.java

```diff
@@ -45,45 +45,50 @@
     }
 
     /**
-     * @param type
+     * @param type explore or configure
      * @return A list of tools or an empty list.
      */
-    public List<ExternalTool> findByType(Type type) {
-        return findByType(type, null);
+    public List<ExternalTool> findDatasetToolsByType(Type type) {
+        String nullContentType = null;
+        return findByScopeTypeAndContentType(ExternalTool.Scope.DATASET, type, nullContentType);
```

Comment:
you could just pass null in the last argument; it's fairly self explanatory

Reply:
Calling the variable "nullContentType" was a deliberate choice to improve readability of the code. We can agree to disagree. 😄